smartsim/_core/mli/comm/channel/dragon_fli.py

```diff
@@ -118,10 +136,13 @@ def from_descriptor(
         :returns: An attached DragonFLIChannel
         :raises SmartSimError: If creation of DragonFLIChanenel fails
```

Comment:
It looks like this also raises a ValueError

Reply:
good eye. added